tests/datastreams/test_processor.py

```diff
@@ -52,3 +54,19 @@ def test_kafka_offset_monitoring():
     assert processor._buckets[bucket_time_ns].latest_produce_offsets[PartitionKey("topic1", 1)] == 34
     assert processor._buckets[bucket_time_ns].latest_produce_offsets[PartitionKey("topic1", 2)] == 10
     assert processor._buckets[bucket_time_ns].latest_commit_offsets[ConsumerPartitionKey("group1", "topic1", 1)] == 14
+
+
+def test_processor_atexit(ddtrace_run_python_code_in_subprocess):
```

Comment:
Was this test supposed to use the `ddtrace_run_python_code_in_subprocess` fixture?

Reply:
I think we should still have a subprocess test to ensure _atexit it actually called when a process shutsdown

Reply:
changed to run an actual subprocess and assert its called